internal/pkg/jsonnet/placeholder.go

```diff
@@ -0,0 +1,177 @@
+package jsonnet
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/google/go-jsonnet/ast"
+	"github.com/umisama/go-regexpcache"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/json"
+	"github.com/keboola/keboola-as-code/third_party/jsonnet/lib/parser"
+)
+
+const (
+	placeholderStart = "<<~~func:"
+	placeholderEnd   = "~~>>"
+	configIdFunc     = "ConfigId"
+	configRowIdFunc  = "ConfigRowId"
+)
+
+// nolint: gochecknoglobals
+var placeholderRegexp = regexpcache.MustCompile(fmt.Sprintf("%s([^:]+):([^~]+)%s",
+	regexp.QuoteMeta(placeholderStart),
+	regexp.QuoteMeta(placeholderEnd),
+))
+
+// ConfigIdPlaceholder generates <<~~func:ConfigId:["<ID>"]~~>>.
+func ConfigIdPlaceholder(id string) string {
+	return functionCallPlaceholder(configIdFunc, id)
+}
+
+// ConfigRowIdPlaceholder generates <<~~func:ConfigRowId:["<ID>"]~~>>.
+func ConfigRowIdPlaceholder(id string) string {
+	return functionCallPlaceholder(configRowIdFunc, id)
+}
+
+// functionCallPlaceholder generates <<~~func:<FUNC_NAME>>:["<ARG1>","<ARG2>"]~~>>.
+func functionCallPlaceholder(funcName string, args ...interface{}) string {
+	return fmt.Sprintf(`%s%s:%s%s`, placeholderStart, funcName, json.MustEncode(args, false), placeholderEnd)
+}
```

Comment:
Spravil som to vseobecne, a je mozne definovat placeholder pre lubovolnu funkciu, s lubovolnym poctom skalarnych argumentov.

Pouzijem to aj pri vstupoch od uzivatela, ktore sa v `JsonNet` budu pouzivat ako `UserInput("mysql-password")`.